### PR TITLE
Create PRS13975BadaMaryam.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
Created new Person ID for ቂሰ፡ ገበዝ፡ ባዕደ፡ ማርያም፡ - ቂሰ፡ ገበዝ supposed to be a title in the church hierarchy.

### CODE REVIEW

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13975BadaMaryam.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ባዕደ፡ ማርያም፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic"/>
+                    <birth/>
+                    <death/>
+                    <floruit>1728</floruit>
```
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS13975BadaMaryam.xml:54]: Done.

File: new/PRS13975BadaMaryam.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            <change who="CH" when="2023-02-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ባዕደ፡ ማርያም፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <faith type="EOTC"/>
```
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS13975BadaMaryam.xml:50]: Done: <name role="qesa gabaz">ቂሰ፡ ገበዝ፡</name>

File: new/PRS13975BadaMaryam.xml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                    <occupation type="ecclesiastic"/>
+                    <birth/>
+                    <death/>
+                    <floruit>1728</floruit>
+                </person>
+            </listPerson><!---->
```
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS13975BadaMaryam.xml:56]: None of that. He is mentioned in the colophon as being in charge when the manuscript (part1) was written. See: <colophon xml:id="coloph2"><locus from="122ra" to="122rb" xml:lang="gez"/>ተጽሕፈት፡ ዛቲ፡ መጽሐፍ፡ በዘመነ፡ ዮሐንስ፡ 
                                ወንጌላዊ፡ በ፸፻ወ፪፻፡ ፳ወ፰፡ ዓመተ፡ ዓለም፡ ወሠርቀ፡ ሌሊት፡ ፬፡ ወሠርቀ፡ መዓልት፡ ፳ወ፮፡ በዘመነ፡ ንጉሥነ፡ <persName ref="PRS2377Bakaffa">በካፋ፡</persName> 
                                ወጳጳስነ፡ <persName ref="PRS6072Krestodo">ክርስቶዶሉ፡</persName> ወመምሕርነሂ፡ <persName ref="PRS00000">መቃርዮስ፡</persName> 
                                ወቂሰ፡ ገበዝሂ፡ <persName ref="PRS00000">በዕዳ፡ ማርያም፡</persName> ወጸሐፊሃኒ፡ <persName role="scribe" ref="PRS13968Arsanyos">አርሳንዮስ፡</persName>
                                ዘተድባበ፡ በመንግሥተ፡ ሰማያት፡ አሜን። 
                                ዛቲ፡ መጽሐፍ፡ ተወጽነት፡ አመ፡ ረቡዑ፡ ለሐምሌ፡ ወተፈጸመት፡ በካልአይ፡ አመት፡ አመ፡ ፳ሀ፡ ለሕዳር፡ በዘመነ፡ <persName ref="PRS00000">ማቴዎስ</persName>።</colophon>

File: new/PRS13975BadaMaryam.xml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        <revisionDesc>
+            <change who="CH" when="2023-02-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ባዕደ፡ ማርያም፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
```
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS13975BadaMaryam.xml:49]: l.  49: <persName xml:lang="gez" type="normalized" corresp="#n1">Bāʿda Māryām</persName>

File: new/PRS13975BadaMaryam.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/PRS13975BadaMaryam.xml:56]: በዕዳ፡ ማርያም፡ is a senior ecclesiastic (qesa gabaz) of the church Tadbaba Maryam,` LOC5843Tadbab` , and mamher Maqaryos must be the abbot.

[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS13975BadaMaryam.xml:56]: Do you think the name must be በዕዳ፡ ማርያም፡ ? In the manuscript it rather looks like ባዕደ፡

[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS13975BadaMaryam.xml:56]: How can I note the church Tadbaba Maryam? Where is this information from?

[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS13975BadaMaryam.xml:56]: How can I note this information?

File: new/PRS13975BadaMaryam.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS13975BadaMaryam.xml:49]: please fill it in the XML tag, not here.

File: new/PRS13975BadaMaryam.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS13975BadaMaryam.xml:56]: The information (as @DenisNosnitsin1970 has provided) must be inserted in the `floruit `tag, it is very important to always provide as much data as possible for disambiguation (he is not the only Baeda Maryam in Ethiopia)

[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS13975BadaMaryam.xml:60]: One can also mention (here, or marking up the colophon) that he was a contemporary of Bakaffa and Krestodolu III, even if this is clear from the date

File: new/PRS13975BadaMaryam.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS13975BadaMaryam.xml:51]: please check the Guidelines https://betamasaheft.eu/Guidelines/?q=roleName&id=roles on marking up `roleName`, we do not use the tag `name `at all

[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS13975BadaMaryam.xml:51]: It should be `<persName><roleName type="office">ቂሰ፡ ገበዝ፡</roleName></persName>`

[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS13975BadaMaryam.xml:51]: Please always use our Guidelines which have examples 
image

and/or search other records
e.g.
image


 if you are not sure how things must be encoded, Creative encoding may damage the application.
Thank you!

File: new/PRS13975BadaMaryam.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS13975BadaMaryam.xml:59]: The information must still be inserted

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13975BadaMaryam.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS13975BadaMaryam.xml:59]: Ok I see it now, thank you!

File: new/PRS13975BadaMaryam.xml [hunk 5]
```diff
@@ -49,13 +49,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                    <occupation type="ecclesiastic">Senior ecclesiastic (qesa gabaz) of the church of 
-                        <placeName ref="LOC5843">Tadbaba Maryam</placeName> in the time of abbot mamher 
-                        <persName ref="PRS0000Maqaryos">Maqaryos</persName>.
-                            </occupation>
+                    <occupation type="ecclesiastic"></occupation>
                     <birth/>
                     <death/>
-                    <floruit when="1728"/>
+                    <floruit when="1728">Senior ecclesiastic (qesa gabaz) of the church of 
+                        <placeName ref="LOC5843">Tadbaba Maryam</placeName> in the time of abbot mamher
```
Comment (eu-genia) [inline new/PRS13975BadaMaryam.xml:56]: please provide the exact complete ID for Tadbaba Maryam

please transcribe mamher

**Comment by CarstenHoffmannMarburg** [comment]
Don't understand how to insert roleName.

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13975BadaMaryam.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS13975BadaMaryam.xml:59]: Thank you, note that Denis has agreed that adding the information on being a contemporary of Bakaffa and Krestodolu III would also be helpful.

File: new/PRS13975BadaMaryam.xml [hunk 6]
```diff
@@ -53,8 +53,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <birth/>
                     <death/>
                     <floruit when="1728">Senior ecclesiastic (qesa gabaz) of the church of 
-                        <placeName ref="LOC5843">Tadbaba Maryam</placeName> in the time of abbot mamher 
-                        <persName ref="PRS0000Maqaryos">Maqaryos</persName>.</floruit>
+                        <placeName ref="LOC5843Tadbab">Tadbaba Maryam</placeName> in the time of abbot mamhər
```
Comment (eu-genia) [inline new/PRS13975BadaMaryam.xml:56]: Māryām

File: new/PRS13975BadaMaryam.xml [hunk 7]
```diff
@@ -53,8 +53,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <birth/>
```
Comment (eu-genia) [inline new/PRS13975BadaMaryam.xml:53]: please correct the roleName  (just copypaste from my earlier comment)

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by CarstenHoffmannMarburg** [comment]
element "roleName" not allowed here.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**